src/data-types/mailstream_cfstream.c

```diff
@@ -995,7 +996,8 @@ int mailstream_cfstream_set_ssl_enabled(mailstream * s, int ssl_enabled)
         count = SecTrustGetCertificateCount(secTrust);
         CFRelease(secTrust);
     }
-    else {
+    // This is work around for Mac OS X 10.6.
+    if(!secTrust || count == 0)  {
```

Comment:
I add debug write and check on Mac OS X 10.6.8.

https://github.com/yamori813/libetpan/commit/d41600982e557e4d6634f741da4545bf5976a9b0

This is result.

bash-3.2$ ./imap-sample xxx@gmail.com password
CFReadStreamCopyProperty kCFStreamPropertySSLPeerTrust 0
CFReadStreamCopyProperty kCFStreamPropertySSLPeerTrust 0
CFReadStreamCopyProperty kCFStreamPropertySSLPeerTrust 0
CFReadStreamCopyProperty kCFStreamPropertySSLPeerTrust 0
CFReadStreamCopyProperty kCFStreamPropertySSLPeerTrust 0
CFReadStreamCopyProperty kCFStreamPropertySSLPeerTrust 0

back to 10.6 work aound.

https://github.com/yamori813/libetpan/commit/2ed0eef2187e1dbe135375ebd09f76a3d4223d21

This is result.

bash-3.2$ ./imap-sample xxx@gmail.com password
CFReadStreamCopyProperty kCFStreamPropertySSLPeerTrust 0
CFReadStreamCopyProperty kCFStreamPropertySSLPeerCertificates 3
connect: 2